AK/Buffered.h

```diff
@@ -108,6 +108,10 @@ class Buffered<StreamType, Size, typename EnableIf<IsBaseOf<InputStream, StreamT
         return true;
     }
 
+    size_t buffered() const { return m_buffered; }
+    // Reading from the stream returned here will most definitely brick the buffering behavior of Buffered.
+    StreamType& underlying_stream() const { return m_stream; }
```

Comment:
wait how does this not explode? you're returning a non-const reference to a non-mutable object from a const function...

Reply:
idk but let's make it non-const